### DISCUSSION TITLE
Webpage organization

### DISCUSSION THREAD

**Discussion by mforets** [JuliaReach/JuliaReach-website#14] [General]
This is to discuss the JuliaReach webpage. Idea for sections:

 (UPDATED)

- Home
- Software
- What's new
- Publications
- Knowledgebase
- ~~Gallery~~
- Team

**Comment by schillic** [comment]
Suggestion:
* Publications (involving JuliaReach)
   I think it is useful to have them in a central place. Then we can link there from the other packages and it is also useful for discussions to have a quick reference. Currently the literature is collected in the individual packages.

**Reply by mforets** [reply]
sounds good

**Reply by mforets** [reply]
we added those sections today, you can check the current website structure at:  https://juliareach.github.io/

**Reply by mforets** [reply]
the  idea we discussed with @SebastianGuadalupe  for the "hero" (currently the green section on top) is to do it on the line of https://biojulia.net/#hero , with a figure showing a flowpipe. maybe add a carousel with several pictures.
